### PR TITLE
Update codewoners with cloud dashboard & authentication

### PR DESCRIPTION
**Pull request by PabloBuchu**
### Pull Request Description

Since the cloud dashboard is now embeded in Enso repository this PR adds owners over dashboard libraries. It also adds eslint specific files owner over @ somebody1234

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [ ] Unit tests have been written where possible.
  - [ ] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.

### CODE REVIEW

**Comment by PabloBuchu** [comment]
@MichaelMauderer @kazcw you guys still want to be call to every PR that touches `/app/ide-desktop/` files?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -39,3 +39,14 @@ Cargo.toml
...
+
+# Cloud Dashboard & Authentication
+/app/ide-desktop/dashboard @PabloBuchu @indiv0 @somebody1234
+/app/ide-desktop/content @PabloBuchu @indiv0 @somebody1234
+/app/ide-desktop/client @PabloBuchu @indiv0 @somebody1234
+/app/ide-desktop/types @PabloBuchu @indiv0 @somebody1234
+/app/ide-desktop/common @PabloBuchu @indiv0 @somebody1234
+
+# Eslint configuration
+/app/ide-desktop/esbuild-plugin-copy-directories @somebody1234
```
Comment (somebody1234) [inline .github/CODEOWNERS:51]: this is an esbuild plugin, not eslint configuration

**Review by somebody1234 — APPROVED** [review]

**Comment by somebody1234** [comment]
entries for `/app/ide-desktop/content-config`, and `/app/ide-desktop/icons` are missing, but to be fair i don't think that's an area the cloud team touches

an entry for `/app/ide-desktop/utils.ts` is also missing

if the entry for the entire directory (`/app/ide-desktop/`) is removed, i think the above entries should be added, not sure who should manage those though

**Comment by PabloBuchu** [comment]
> entries for `/app/ide-desktop/content-config`, and `/app/ide-desktop/icons` are missing, but to be fair i don't think that's an area the cloud team touches

as you said its not the cloud are and it will be take by the precedence from `/app/ide-desktop/` this way I didn't want to remove this entry

> 
> an entry for `/app/ide-desktop/utils.ts` is also missing

This should also goes to eslint configuration right?

> 
> if the entry for the entire directory (`/app/ide-desktop/`) is removed, i think the above entries should be added, not sure who should manage those though

as above. left it intentionally so at least wdanilo will be the owner

**Comment by somebody1234** [comment]
`utils.ts` is for build scripts (`esbuild`), as is `esbuild-plugin-copy-directories`

`eslint.config.js` is for the linter (`eslint`)

**Review by wdanilo — APPROVED** [review]

**Comment by kazcw** [comment]
> @MichaelMauderer @kazcw you guys still want to be call to every PR that touches `/app/ide-desktop/` files?

No, I'd be happy not to

**Pull request merged**